src/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Client diversity
+description: A high level explanation of the importance of Ethereum client diversity.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+
+The behaviour of an Ethereum node is controlled by the client software it runs. There are several production-level Ethereum clients, each one developed and maintained in different languages by separate teams. The clients are built to a common spec that ensures the clients seamlessly communicate with each other and have the same functionality and provide an equivalent user experience. However, at the moment the distribution of clients across nodes is not even enough to realise this network fortification to its full potential. Ideally, users divide roughly equally across the various clients to bring as much client diversity as possible to the network.
+
+## Prerequisites {#prerequisites}
+
+If you don't already have an understanding of what nodes and clients are, check out [Nodes and clients](/developers/docs/nodes-and-clients/). The Beacon Chain is explained [here](/src/content/eth2/beacon-chain/index.md). Execution and consensus layers are defined in the [glossary](/src/content/glossary/index.md).
+
+## Why are there multiple clients?
+
+Multiple, independently developed and maintained clients exist because client diversity makes the network more resilient to attacks and bugs. Multiple clients is a strength unique to Ethereum - other blockchains rely on the infallibility of a single client. However, it is not enough simply to have multipe, clients available, they have to be adopted by the community and the total active nodes distributed relatively evenly across them.
```

Comment:
To emphasize the point of "why", may be worth making a general note here that if a network has one client and there is a bug, this could be catastrophic, while a bug in one client out of many would likely be insignificant.